nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -568,4 +585,15 @@ public Task fetchTaskById(String documentId) {
         return (Task) getResponse(documentId);
     }
 
+    private void refuseIfDateFound(Object obj) {
+        if (obj instanceof Calendar || obj instanceof Date) {
+            throw new IllegalArgumentException(MSG_DATE_UNSUPPORTED);
+        }
+        if (obj instanceof Map) {
```

Comment:
You're totally right. That's why I didn't want the support the check at the beginning, it requires so many work... :broken_heart: So, should we support other `Collection` or `Iterable` and array too? It there any possibility that I forget?